benchmark/README.md

```diff
@@ -73,24 +80,10 @@ stack build
 Run the benchmark using the compiled binary:
 
 ```
-./benchmark.py haskell/.stack-work/dist/*/Cabal-*/build/ac-bench/ac-bench --prefix haskell
```

Comment:
Was there a problem with this approach? It looks like the intention was that it's possible to just  copy&paste this command, but I presume that breaks when there's more than one match?

Reply:
Yes, the Nix config broke this (previously pasteable) line.